components/scripts/lib/build-scan-online.sh

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 
-readonly FETCH_BUILD_SCAN_DATA_JAR="${LIB_DIR}/build-scan-clients/fetch-build-scan-data-cmdline-tool-${SCRIPT_VERSION}-all.jar"
+readonly FETCH_BUILD_SCAN_DATA_JAR="${SCRIPT_DIR}/lib/develocity/build-scan-summary-${SUMMARY_VERSION}.jar"
```

Comment:
Should we call this `BUILD_SCAN_SUMMARY_JAR` now?

Reply:
Done